pages/common/npx.md

```diff
@@ -0,0 +1,15 @@
+# npx
+
+> Execute binaries from `npm` packages.
+
+- Execute the binary from a given npm module:
+
+`npx {{module_name}}`
+
+- In case a package has multiple binaries, specify the package name along with the binary:
```

Comment:
What is meant by a package name here?

Reply:
The name of the npm package. Usually the package name is the same as the binary name. Sometimes, a package has multiple binaries, or the binary name is different from the package name. In that case, we need to differentiate between the two.

Reply:
So,it's ok ?

Reply:
Yes, I believe so. Unless @sbrl wants to clarify something further in the description.

Reply:
ok

Reply:
Ah, ok. Maybe it was the `module_name` that threw me because that's actually the binary name.